scripts/install_third_party_libs.py

```diff
@@ -174,6 +167,14 @@ def compile_protobuf_files(proto_files_paths):
             python_utils.PRINT(stderr)
             raise Exception('Error compiling proto files at %s' % path)
 
+    compiled_protobuf_dir = (
+        pathlib.Path(os.path.join(common.CURR_DIR, 'proto_files')))
+    for p in compiled_protobuf_dir.iterdir():
+        if p.suffix == '.py':
+            common.inplace_replace_file(
```

Comment:
Perhaps you could add a comment explaining what exactly this is doing.

Reply:
Added.